webpack.config.js

```diff
@@ -0,0 +1,32 @@
+var path = require('path');
```

Comment:
Can we use es6 for consistency?

Example:
`import path from 'path';`

`export default ...`

Reply:
I think I tried using es6 in that file and it was complaining. I'll get a SS later on when i'm on my mac, maybe you know the solution